BuildResidentialHPXML/measure.rb

```diff
@@ -3944,6 +3971,18 @@ def self.unavailable_period_exists(hpxml, column_name, begin_month, begin_day, b
     return false
   end
 
+  # Set header properties, including:
+  # - vacancy periods
+  # - power outage periods
+  # - software info program
+  # - simulation control
+  # - emissions scenarios
+  # - utility bill scenarios
+  #
+  # @param runner [OpenStudio::Measure::OSRunner] OpenStudio Runner object
+  # @param hpxml [HPXML] HPXML object
+  # @param args [Json::Value] a Json::Value hash
```

Comment:
I don't really understand this either; `args.class` says it's simply a hash.

```suggestion
  # @param args [Hash] Map of :argument_name => value
```

Reply:
Oops. I was copying the C++ types directly out of the SDK. Good catch.